Tests/IntegrationTests/IntegrationTests.swift

```diff
@@ -94,6 +94,9 @@ public class IntegrationTests: XCTestCase {
         if let result = ret.0 {
           if let r = result.nextResult() {
             XCTAssertEqual(1, r["Id"] as! Int)
+            XCTAssertEqual("Audi", r["Name"] as! String)
+            XCTAssertNotNil(r["Price"])
+            XCTAssertNotNil(r["UpdatedAt"])
```

Comment:
added these to double check all fields are present. the new assertion for the TEXT type is working :)